lib/hmmlearn/base.py

```diff
@@ -139,8 +139,9 @@ class _BaseHMM(BaseEstimator):
 
     def __init__(self, n_components=1,
                  startprob_prior=1.0, transmat_prior=1.0,
-                 algorithm="viterbi", random_state=None,
-                 n_iter=10, tol=1e-2, verbose=False,
+                 algorithm="viterbi", implementation="log",
```

Comment:
Technically, this is a backwards incompatible change (in case someone passed all the arguments positionally).  I agree it is most unlikely to affect anyone, but if we're going to break this, we may as well make all args (say, starting from startprob_prior) keyword-only, and add a note in the changelog.  (The whole things needs a changelog entry as well, anyways.)

Reply:
This should be fixed now.